gradle/maven.gradle

```diff
@@ -81,8 +95,8 @@ subprojects {
 
           licenses {
             license {
-              name = 'Copyright 2019 Linkedin, All Rights Reserved'
-              url = 'https://github.com/linkedin/brooklin/blob/master/LICENSE'
+              name = 'Copyright 2021 Linkedin, All Rights Reserved'
```

Comment:
copyright on new files are from the year they are added, but for the repo it is from 2019 itself